spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1154,6 +1139,87 @@
             )
           end
         end
+
+        context 'sms notifications enabled' do
+          let(:pending_enrollment) do
+            create(
+              :in_person_enrollment,
+              :pending,
+              :with_notification_phone_configuration,
+              capture_secondary_id_enabled: true,
+            )
+          end
+
+          before do
+            allow(IdentityConfig.store).to receive(:in_person_send_proofing_notifications_enabled).
+              and_return(true)
+          end
+
+          context 'enrollment is expired' do
+            it 'deletes the notification phone configuration without sending an sms' do
+              stub_request_expired_proofing_results
+
+              expect(pending_enrollment.notification_phone_configuration).to_not be_nil
+
+              job.perform(Time.zone.now)
+
+              expect(pending_enrollment.reload.notification_phone_configuration).to be_nil
+              expect(pending_enrollment.notification_sent_at).to be_nil
+            end
+          end
+
+          context 'enrollment has passed proofing' do
+            it 'invokes the SendProofingNotificationJob for the enrollment' do
+              stub_request_passed_proofing_results
+
+              expect(pending_enrollment.notification_phone_configuration).to_not be_nil
+              expect(pending_enrollment.notification_sent_at).to be_nil
+
+              expect { job.perform(Time.zone.now) }.
+                to have_enqueued_job(InPerson::SendProofingNotificationJob).
+                with(pending_enrollment.id)
+            end
+          end
+
+          context 'enrollment has failed proofing' do
+            it 'invokes the SendProofingNotificationJob for the enrollment' do
+              stub_request_failed_proofing_results
+
+              expect(pending_enrollment.notification_phone_configuration).to_not be_nil
+              expect(pending_enrollment.notification_sent_at).to be_nil
+
+              expect { job.perform(Time.zone.now) }.
+                to have_enqueued_job(InPerson::SendProofingNotificationJob).
+                with(pending_enrollment.id)
```

Comment:
I noticed that these aren't testing the `wait_until` param, but that's probably fine since it should be covered by #8848.